<?xml version="1.0"?>

<launch>

  <arg name="n" value="20" />

  <node name="spawn_balls" pkg="leo_sim" type="spawn_balls.py" args="20" output="screen">
    <param name="model_path" value="$(find tennis-description)/models/tennis_ball/model.sdf" />
  </node>

</launch>
